src/eth.rs

```diff
@@ -3,7 +3,7 @@ use core::fmt::Debug;
 use core::ptr;
 use core::time::Duration;
 
-use ::log::*;
```

Comment:
Why?

Reply:
I have rust format active on save and it changed it. I'll change it back.